test/eegpsr_grammar.fcfg

```diff
@@ -0,0 +1,175 @@
+T[{actions : <A1>}] -> C[A1]
```

Comment:
Variables can be as long as we want, right? Please make give variarables like "A" a clearer name